packages/perseus/src/util/parse-perseus-json/perseus-parsers/perseus-image-background.ts

```diff
@@ -10,8 +10,8 @@ import {
 import {convert} from "../general-purpose-parsers/convert";
 import {stringToNumber} from "../general-purpose-parsers/string-to-number";
 
+import type {PerseusImageBackground} from "../../../perseus-types";
 import type {Parser} from "../parser-types";
-import type {PerseusImageBackground} from "@khanacademy/perseus";
```

Comment:
I've seen "self" imports like this sneak into our codebase before. I _think_ they work, but it'd be nice if we could lint for them and prevent them.

Reply:
I've noticed this too. I thought we have a lint rule, but maybe it's not working for TS?